lib/rspec/matchers/built_in/base_matcher.rb

```diff
@@ -126,22 +126,6 @@ def self.underscore(camel_cased_word)
 
       private
```

Comment:
```suggestion
```

A agree with RuboCop here, there are no method definitions below (anymore), and this access modifier is useless and can be removed.

My apologies, I’m from mobile and couldn’t mark two lines, the suggestion won’t apply clearly.